jwst/extract_1d/extract.py

```diff
@@ -115,13 +116,23 @@ def open_extract1d_ref(refname: str) -> dict:
         If the extract1d reference file is in JSON format, ref_dict will be the
         dictionary returned by json.load(), except that the file type
         ('JSON') will also be included with key 'ref_file_type'.
+        If the extract1d reference file is in asdf format, the ref_dict will
+        be a containing two keys: ref_dict['ref_file_type'] = 'ASDF'
+        and ref_dict['ref_model'].
         If the reference file is an image, ref_dict will be a
         dictionary with two keys:  ref_dict['ref_file_type'] = 'IMAGE'
         and ref_dict['ref_model'].  The latter will be the open file
         handle for the jwst.datamodels object for the extract1d file.
     """
+
     if refname == "N/A":
         ref_dict = None
+    elif exptype == 'MIR_MRS' or exptype == 'NRS_IFU':
```

Comment:
Doesn't affect functionality, but you could condense this to:
`elif exptype in ['MIR_MRS', 'NRS_IFU']:`